src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/GuardAuthenticationFactory.php

```diff
@@ -111,11 +113,6 @@ public function createAuthenticator(ContainerBuilder $container, string $firewal
         return $authenticatorIds;
     }
 
-    public function createEntryPoint(ContainerBuilder $container, string $id, array $config, ?string $defaultEntryPointId): string
-    {
-        return $this->determineEntryPoint($defaultEntryPointId, $config);
-    }
```

Comment:
This causes a test failure:

> Error: Call to undefined method Symfony\Bundle\SecurityBundle\DependencyInjection\Security\Factory\GuardAuthenticationFactory::createEntryPoint()
/home/travis/build/symfony/symfony/src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/Security/Factory/GuardAuthenticationFactoryTest.php:181

But you introduced that test in your other PR, so it just needs to be updated I think.